### PR TITLE
IPS on DCC KB

### PR DESCRIPTION
**Pull request by CMeyer88**
Adding KB for IPS on DCC

### CODE REVIEW

File: Security/IPS-on-DCC.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+{{{ "title": "IPS - Centurylink Dedicated Cloud Compute (DCC) FAQ",
```
Comment (srveit) [inline Security/IPS-on-DCC.md]: Please reformat this for consistency:

```
{{{
  "title": "IPS - Centurylink Dedicated Cloud Compute (DCC) FAQ",
  "date": "02-22-2016",
  "author": "Client-Security",
  "attachments": [],
  "contentIsHTML": false,
  "sticky": false
}}}
```

File: Security/IPS-on-DCC.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        "author": "Client-Security",
+        "attachments": [],
+        "contentIsHTML": false,
+        "sticky": false }}}
+        
+### FAQ for Installing and Implementing IPS Anywhere for CenturyLink Dedicated Cloud Compute (DCC)
+        
+This document is intended to summarize (aka Quick Start) the installation and notification configuration steps for CLC’s IPS “Anywhere” service as derived from the services public KBs.  
+        
+## Pre-Requisites
```
Comment (srveit) [inline Security/IPS-on-DCC.md]: Change to `###`

File: Security/IPS-on-DCC.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+## Pre-Requisites 
+  * Compatible server OS 
+    * IPS supports all Managed DCC OS Images at this time
+    * Supported OSs KB: https://www.ctl.io/knowledge-base/security/supported-ips-oses/
+  * Firewall Rules
+  * Centurylink Cloud Account
+
+KB: https://www.ctl.io/knowledge-base/security/getting-started-with-ips/#prerequisites 
+        
+## Installing IPS on Target Server
```
Comment (srveit) [inline Security/IPS-on-DCC.md]: Change to `###`

File: Security/IPS-on-DCC.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+Run below command in a terminal session on target server:
+  * Linux-based Operating Systems
+> * curl https://api.client-security.ctl.io/ips/scripts/install.sh | sudo CLC_USERNAME=<your.clc.username> CLC_PASSWORD=<your.clc.password> CLOUD_PROVIDER= CTL_DCC bash
+
+  * Windows-based Operating Systems
+> * Set-ExecutionPolicy -ExecutionPolicy RemoteSigned -Scope Process
+> * (New-Object System.Net.WebClient).DownloadFile("https://api.ts.client-security.ctl.io/ips/scripts/install.ps1","$env:temp\install.ps1") ; cd $env:temp ; .\install.ps1 -controlUser "<your.clc.account>" -controlUserPassword "<your.clc.password>" -accountAlias "<your.clc.account,alias>" -cloudProvider "CTL_DCC"
+> * Note: for the user input, you will need to enclose the data in “”.  For example, if your username was testuser, for the option of -controlUser "<your.clc.account>", you would use -controlUser "test user”.
+
+## Notifications
```
Comment (srveit) [inline Security/IPS-on-DCC.md]: Change to `###`

File: Security/IPS-on-DCC.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+  * The servers to update the notification on
+  * The email addresses or email addresses to send notifications to. 
+  * The WebHook URL to send notifications to.  NOTE: There can only be 1 WebHook setup.
+  * The syslog server to forward event data to.  NOTE: There can only be 1 syslog server setup.
+
+The full documentation for the IPS API is documented in the KB link listed below.  Examples of how to request each of the notification mediums (Email, Slack, Webhook, & Syslog) are reviewed.   
+
+  * IPS API KB: https://www.ctl.io/knowledge-base/security/ips-api/
+
+## Additional KBs and Links
```
Comment (srveit) [inline Security/IPS-on-DCC.md]: Change to `###`

**Comment by CMeyer88** [comment]
Updated for all headers to be ###.

**Comment by CMeyer88** [comment]
Corrected Header Issue

**Pull request merged**